blueocean-core-js/src/js/jwt.js

```diff
@@ -41,7 +41,8 @@ export default {
      */
     verifyToken(token, certObject) {
         return new Promise((resolve, reject) =>
-            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg] }, (err, payload) => {
+            // Allow the clocks to be 60s out of sync.
+            jwt.verify(token, jwk2pem(certObject), { algorithms: [certObject.alg], clockTolerance: 60 }, (err, payload) => {
```

Comment:
@tfennelly this looks fine to me except it could be smaller number.  The tolerance is entirely what client decides. Only thing server sends is nbf (not before), that is time before it can't be used, default is 30 sec earlier than the time it was granted (default leeway to account for client skew).